src/tests/map/all.mk

```diff
@@ -19,11 +19,17 @@ $(MAP_OUTPUT): $(MAP_UNIT_BIN) | $(BUILD_DIR)/tests/map/
 #
 $(BUILD_DIR)/tests/map/%.out: $(top_srcdir)/src/tests/map/%
 	${Q}echo MAP_TEST $(notdir $<)
-	${Q}if ! $(MAP_UNIT) -d $(top_srcdir)/raddb -D $(top_srcdir)/share/dictionary $< > $@ 2>&1; then \
+	${Q}rm -f "$@.ok"
```

Comment:
You don't need to remove the .ok files as this will only be invoked in the .ok file doesn't exist

Reply:
If I don't remove the *.ok and the $(MAP_UNIT) fails, then it will rise a false positive.

Reply:
The solution here is to update `unit_test_map` and `unit_test_attribute` to write those files before they exit.  I'll take a look at it.

Reply:
See below for the comment on having those utilities remove the ok files when they start